### ISSUE TITLE
Difference between DL method and conventional SI

### ISSUE THREAD

**Issue by toghrul97** [RhFeng/UN_PORO#2]
1. Could you please tell, what is the difference between this supposed method and conventional seismic inversion method. On the paper there was a quote 'Figure 4a shows the predicted porosity by the proposed deep learning method. For a comparison, a conventional poststack seismic inversion is used to invert AI, which is later converted to porosity according to equation 3 (Figure 4b). A model-driven optimization is utilized, in which the initial background AI model is perturbed iteratively until the derived synthetic seismic data best fit the real data or until a preset number of iterations is reached'. But in the model synthetic data is created by the same way by showing the same relationship between AI and porosity.   Loss function is targeted to minimize the difference between synthetic and true data. In SI we would try to do the same. The only difference I see is us trying to make absolute porosity closer to prior porosity. Is it the difference maker?  Does it make the prediction better than SI?
2. Is the synthetic data fully available? I would like to create whole 2d section if the data is available.
3. What does the error limit mean on the paper? It was written that it should be 0.001? Should loss function be less than 0.001, or it mean something else? In case loss should be less than 0.001, why CC is not near 1? At loss near 0.2, I got CC of 0.88.
4. Tensorflow 1 is  not available anymore. To run the model we can use this code: !tf_upgrade_v2 --infile un_poro.py --outfile un_poro_tf2.py

Thank you in advance. I am looking forward for your answers. Hope you find them interesting :)

**Comment by RhFeng** [comment]
1. In the typical porosity prediction, the impedance is inverted first, then the inverted impedance is converted to porosity based on some empirical equation. In the proposed deep learning method, this process is integrated into one step, even though the same empirical equation is going to be used for the conversion of porosity based on impedance.
2. The whole synthetic data will be made available soon, as I have to dig into the hard disk to locate it.
3. In the real case study, the error limit is changing along the CMP locations, based equation (7). It is intially set as 0.001, then the CC is considered, it changes at another location. See figure 12a, the error limit should be smaller than or equal to 0.001, as  CC<=1
4. Thank you.

**Comment by RhFeng** [comment]
1. In the typical porosity prediction, the impedance is inverted first, then the inverted impedance is converted to porosity based on some empirical equation. In the proposed deep learning method, this process is integrated into one step, even though the same empirical equation is going to be used for the conversion of porosity based on impedance.
2. The whole synthetic data will be made available soon, as I have to dig into the hard disk to locate it.
3. In the real case study, the error limit is changing along the CMP locations, based equation (7). It is intially set as 0.001, then the CC is considered, it changes at another location. See figure 12a, the error limit should be smaller than or equal to 0.001, as  CC<=1
4. Thank you.

**Comment by toghrul97** [comment]
Thank you for your answer professor. Now it makes sense. I am waiting impatiently for the synthetic data you used in the paper or any other
